### ISSUE TITLE
Order-independent transparency in instanced layers (polygon/geojson)

### ISSUE THREAD

**Issue by ibgreen** [uber/deck.gl#2918]
#### Target Use case
- nebula.gl is implementing support for editing volumes, and transparency ordering in instanced layers has come up as a major issue.

- @tsherif has several impressive examples that should be considered [depth peelin g](https://tsherif.github.io/webgl2examples/oit-dual-depth-peeling.html)

#### Proposed feature
- Optional support in luma and/or deck to enable order-independent transparency in instanced layers.

#### To Do List 
  - [ ] Add label and assign to milestone 
  - [ ] Coding
  - [ ] Doc update
  - [ ] What’s new update 
  - [ ] Test

**Comment by tsherif** [comment]
What kind of requirements can we put on API support? Both WBOIT and depth peeling require float textures, and are simpler with multiple render targets. 

Can we implement this as WebGL 2-only?

**Comment by ibgreen** [comment]
> Can we implement this as WebGL 2-only?

YES! I think this would be a great case for a WebGL2 only feature. 

I know that nebula.gl is used on a lot of "lower end" machines but I believe this feature is for more advanced apps and does not need to support those @georgios-uber

**Comment by georgios-uber** [comment]
I want to implement **WBOIT** because **depth peeling** will be too slow.
_WebGL 2.0 requirement is ok._
But I want to find a way to make any layer work with it seamlessly, and that could be a challenge.

**Comment by tsherif** [comment]
I agree that depth peeling will likely be too slow for us.

If we want to do it in WebGL 1, this will be important: https://www.cbrebuild.com/nyc/blog/posts/webgl-order-independent-transparency-single-render-target/
